src/macros.rs

```diff
@@ -2,54 +2,76 @@
 /// ```
 /// # use druid_widget_nursery::selectors;
 /// selectors! {
+///     /// Foo the baz
 ///     FOO,
+///     /// Bar the qux yay much
 ///     BAR: usize,
 /// }
 /// ```
 /// expands to
 /// ```
 /// # use druid::Selector;
+/// /// Foo the baz
 /// pub const FOO: Selector = Selector::new("path::to::module::FOO@0:0");
+/// /// Bar the qux yay much
 /// pub const BAR: Selector<usize> = Selector::new("path::to::module::BAR@0:0");
 /// ```
 #[macro_export]
 macro_rules! selectors {
-    ($($name:ident $( : $ty:ty)?),* $(,)?) => {$(
-        pub const $name: ::druid::Selector<$($ty)?> = ::druid::Selector::new(concat!(
-            module_path!(),
-            "::",
-            stringify!($name),
-            "@",
-            line!(),
-            ":",
-            column!()
-        ));
-    )*};
+    (
+        $(
+            $(#[$inner:ident $($args:tt)*])*
```

Comment:
we just use `:meta` here
```suggestion
            $(#[$attr:meta])*
```

Reply:
:+1:
Done.